### DISCUSSION TITLE
help with weird messege

### DISCUSSION THREAD

**Discussion by HolySrulik** [Pryaxis/TShock#2408] [General]
hello, not sure it's the right place, im new to all of this. 
i got this error when my character move and etc. 
help me if you can please :)

![image](https://user-images.githubusercontent.com/87757162/126470149-c7541a1d-651d-4271-a75a-37a1c3bac0d7.png)

**Comment by moisterrific** [comment]
you can ignore if there's nothing unusual going on in-game, that's just debug info

**Reply by HolySrulik** [reply]
Well actually only later I saw that in the game it said you can't build on spawn point in red. Left bottom side.
When I moved and tried to do stuff

**Reply by bartico6** [reply]
Yeah that's because spawn protection is enabled in server settings, disable it
